docs/sources/dashboards/create-reports/_index.md

```diff
@@ -138,35 +201,55 @@ The options are:
 
 | Option | Description |
 | ------ | ----------- |
-| Source dashboard (required) | Select the dashboard from which you want to generate the report. |
-| [Template variables](#template-variables) | Select the variable values for the selected dashboard. This option is only displayed if the dashboard has variables. |
-| [Time range](#time-range) | If you leave the field empty, reports use the saved time range of the dashboard. Optionally, you can change the time range of the report. |
-| Add another dashboard | Add more dashboards to the report. |
+| Source dashboard (required) | Select or update the dashboard from which you want to generate the report. If you've created your report directly from a dashboard, this field is already filled in with the name of the current dashboard. |
+| [Time range](#time-range) | Update the report time range. If the dashboard is preselected, the default time range is that of the dashboard. Otherwise, the default time range is **Last 6 hours**. |
```

Comment:
```suggestion
| [Time range](#time-range) | Update the report time range. If you've created the report directly from a dashboard, the, the default time range is that of the dashboard. Otherwise, the default time range is **Last 6 hours**. |
```